.github/workflows/static_checks_etc.yml

```diff
@@ -59,85 +59,51 @@ jobs:
               - 'tools/**'
               - 'bootstrap.sh'
               - 'misc/git/hooks/asthelpers'
-            end_to_end:
+            docker:
```

Comment:
Were changes to this file intentional?

Reply:
Yes, it was, I thought it made more sense as we are not using any end-to-end tests in this workflow. This filtering block is used for the docker checks.